packages/block-library/src/button/editor.scss

```diff
@@ -33,50 +33,6 @@
 	}
 }
 
-.wp-block-button__inline-link {
-	color: $gray-700;
-	height: 0;
-	overflow: hidden;
-	max-width: 290px;
-
-	&-input__suggestions {
-		max-width: 290px;
-	}
-
-	@include break-medium() {
-		max-width: 260px;
-
-		&-input__suggestions {
-			max-width: 260px;
-		}
-
-	}
-	@include break-large() {
-		max-width: 290px;
-
-		&-input__suggestions {
-			max-width: 290px;
-		}
-
-	}
-
-	.is-selected & {
-		height: auto;
-		overflow: visible;
-		margin-top: $grid-unit-20;
-	}
-}
-
-.wp-button-label__width {
```

Comment:
Introduced here:
https://github.com/WordPress/gutenberg/commit/fb1cc0e8cec5c55c17de72b13a13214beb6fe602#diff-0fd478f79554ac91f10a7020b813e45f77245ddcde593682bab998b2ce2c232cR70

But looks like it was a leftover from some pre-commit WIP since a matching element cannot be found.